apps/boilerplate-front-end/src/config/oktaLogin.js

```diff
@@ -0,0 +1,16 @@
+const CLIENT_ID = import.meta.env.VITE_APP_OKTA_CLIENT_ID
+const ISSUER = import.meta.env.VITE_APP_OKTA_BASE_URL
+const OKTA_TESTING_DISABLEHTTPSCHECK = import.meta.env.VITE_APP_OKTA_TESTING_DISABLEHTTPSCHECK
+const BASENAME = ''
+const REDIRECT_URI = `${window.location.origin}${BASENAME}/login/callback`
+
+export default {
+  oidc: {
+    clientId: CLIENT_ID,
+    issuer: ISSUER,
+    redirectUri: REDIRECT_URI,
+    scopes: ['openid', 'profile', 'email'],
```

Comment:
can you link to docs in comment about L7 so people can understand what scopes do etc.?

Reply:
Done 👍

Reply:
Boom